cfpb/capital-framework
# Convert form units to ems

## Changes

- Convert fixed pixel units to ems in the forms.

## Testing

- Check out this branch and follow https://github.com/cfpb/capital-framework/blob/canary/CONTRIBUTING.md#testing-components-locally
- Check cf-forms.
